Cargo.toml

```diff
@@ -8,5 +8,5 @@ homepage = "https://github.com/sgrif/r2d2-diesel"
 repository = "https://github.com/sgrif/r2d2-diesel"
 
 [dependencies]
-r2d2 = "0.7.0"
+r2d2 = "0.8"
```

Comment:
This didn't change any actual code, just tests and the README. Should this be `>= 0.7.0 < 0.9.0`?

Reply:
Sure, why not